base/js/admin.js

```diff
@@ -175,7 +175,13 @@
             $el.find('.siteorigin-widget-field-repeater-item').sowSetupRepeaterItems();
 
             // Set up any color fields
-            $fields.find('> .siteorigin-widget-input-color').wpColorPicker();
+			$fields.find('> .siteorigin-widget-input-color').wpColorPicker( {
```

Comment:
Indenting on this section seems to have changed to tab instead of space indentation.